src/plugins/identity/README.md

```diff
@@ -10,10 +10,136 @@ GitHub and Discourse (potentially with a different username on each service).
 We would like to combine these two identities together, so that we can
 represent that user's combined cred properly. The Identity plugin enables this.
 
-Specifically, the instance maintainer can provide a (locally unique) username
-for the user, along with a list of aliases the user is known by, e.g.
-`github/username` and `discourse/other_username`. The aliases are simple string
-representations, that are intended to be easy to maintain by hand in a
-configuration file. Then, the identity plugin will provide a list of
-`NodeContraction`s that can be used by `Graph.contractNodes` to combine the
-user identities as described.
+## Status and Caveats
+
+The Identity plugin currently only applies to new (Weighted)Graphs and Cred
```

Comment:
> Therefore, if you re-run SourceCred after merging new identities, the scores will shift. This can be tricky,

This isn't really the issue I'm trying to highlight. Cred scores can shift around for a number of reasons (and this is normal because retroactive Cred is a feature). Contracting identities being one, weight changes being another common case.

The problem that identity contraction has which a weight change doesn't, is that Grain might have been distributed to your GitHub and Discourse node in previous weeks, while post-contraction it would be distributed to your identity node going forward. Not dealing with this you're going from having 2 accounts in the Grain ledger, to 3! While you'd want this to be 1, just the identity one.

I'll take a stab at clarifying this section.